src/Backend/Modules/Pages/Domain/Page/PageRepository.php

```diff
@@ -413,19 +413,25 @@ private function buildGetQuery(int $pageId, int $revisionId, string $language):
             ->leftJoin(PageBlock::class, 'b', Join::WITH, 'b.revisionId = p.revisionId AND b.extraId IS NOT NULL')
             ->leftJoin(ModuleExtra::class, 'e', Join::WITH, 'e.id = b.extraId AND e.type = :type')
             ->where('p.id = :id')
-            ->andWhere('p.revisionId = :revisionId')
-            ->andWhere('p.language = :language')
             ->groupBy('p.revisionId')
         ;
 
-        $qb->setParameters(
-            [
-                'type' => 'block',
-                'id' => $pageId,
-                'revisionId' => $revisionId,
-                'language' => $language,
-            ]
-        );
+        $parameters = [
+            'type' => 'block',
```

Comment:
isn't there already a type for it? `Common\ModuleExtraType`?

Reply:
Changed in a45d5e53